snorkel/classification/utils.py

```diff
@@ -116,3 +117,17 @@ def collect_flow_outputs_by_suffix(
         for flow_name in sorted(flow_dict.keys())
         if flow_name.endswith(suffix)
     ]
+
+
+def metrics_dict_to_dataframe(metrics_dict: Dict[str, float]) -> pd.DataFrame:
```

Comment:
should this have a task field as well? (or I may be remembering an older format of the metric dictionary)

Reply:
good catch, in this case, `task_name == label_name`